prettyPrinters/providers.py

```diff
@@ -283,7 +283,13 @@ def update(self):
         # type: () -> None
         self.length = self.valobj.GetChildMemberWithName("len").GetValueAsUnsigned()
         self.buf = self.valobj.GetChildMemberWithName("buf")
-        self.data_ptr = self.buf.GetChildAtIndex(0).GetChildAtIndex(0).GetChildAtIndex(0)
+
+        self.data_ptr = self.buf.GetChildMemberWithName("ptr").GetChildAtIndex(0)
+        # https://github.com/rust-lang/rust/commit/7a0911528058e87d22ea305695f4047572c5e067
+        # TODO: get rid of it after new rustc stable release
```

Comment:
We can't drop it because we will break debugging with rustc below 1.33
Let's better add `# BACKCOMPAT: rust 1.32 some text` comment to remove this when we will drop rust 1.32 support